src/typescript/azure-openapi-validator/rule.ts

```diff
@@ -7,7 +7,8 @@ import { JsonPath } from "../jsonrpc/types";
 export enum OpenApiTypes {
   "default" = 1 << 0,
   "arm" = 1 << 1,
-  "dataplane" = 1 << 2
+  "dataplane" = 1 << 2,
+  "doc" = 1 << 3,
```

Comment:
why is doc an OpenApiType? The intent of this enum is to indicate what type of spec we're dealing with. So far we know of just a basic regular OpenAPI doc, let's say not related to Azure, and Azure related ones, into to categories, arm and dataplane. Doc doesn't seem to map to these categories.

Reply:
I missed this. We do not want to introduce a new OpenAPI type here, only a new category for the rules themselves.

Reply:
Will be removed. Since these rules apply to all OpenAPI types, I'm assuming that the rules will need to be `|`ed together, rather than set to `default`?

Reply:
That depends on the rule. Given you are dealing with descriptions here I'd say its got to do with arm openapitype, however, confirm this with @kirthik